sormas-backend/src/main/resources/sormas_import_all_countries.csv

```diff
@@ -0,0 +1,202 @@
+## All WHO countries
+"defaultName","externalId","isoCode","unoCode","archived"
```

Comment:
This won't work on systems that use another csv delimiter (e.g. ";").
You will probably have to explicitly define the delimiter when importing this.